GitUI/CommandsDialogs/ScriptOptionsProvider.cs

```diff
@@ -0,0 +1,49 @@
+using GitCommands;
+using GitUI.ScriptsEngine;
+
+namespace GitUI.CommandsDialogs;
+
+internal class ScriptOptionsProvider : IScriptOptionsProvider
+{
+    private const string _selectedFiles = "SelectedFiles";
+    private const string _lineNumber = "LineNumber";
+
+    private Func<IEnumerable<string>?> _getSelectedFiles;
```

Comment:
Enumerations shall not be `nullable`.

Reply:
What is the alternative? (See below)

Reply:
(`Array.Empty` instead of `null` `IEnumerable`.)

Reply:
It's not pretty, but the result, in this case, is identical - returning `null` from `GetValue`. Therefore, accepted.

P.S. When looking for `IEnumerable<string>?` to ensure I removed all of them, I did notice it being used in several other parts of the codebase.